attributions2json.py

```diff
@@ -65,32 +66,33 @@ def get_license_text(repo):
     path = find_license_path(repo)
     if path:
         data = send_content_request(repo, path)
-        return data['content'].decode(data['encoding'])
+        decodedData = base64.b64decode(data['content'])
```

Comment:
Should we still be reading `data['encoding']` or at least assert that it is `"base64"`?

Reply:
Yeah good idea, how about this?
```suggestion
if data['encoding'] != 'base64':
    print("Encoding of Github response was not base64")
    sys.exit(1)
decodedData = base64.b64decode(data['content'])
return decodedData.decode('utf-8')
```

Reply:
Seems good enough for now since we haven't seen any other encoding so far.